config/v2/resolvers.go

```diff
@@ -307,6 +309,28 @@ func ResolveDatadogProvider(commons ...Common) *DatadogProvider {
 	return p
 }
 
+func ResolveSentryProvider(commons ...Common) *SentryProvider {
+	var p *SentryProvider
+	for _, c := range commons {
+		if c.Providers == nil || c.Providers.Datadog == nil {
+			continue
+		}
+		p = c.Providers.Sentry
+	}
+
+	version := lastNonNil(SentryProviderVersionGetter, commons...)
+	baseURL := lastNonNil(SentryProviderBaseURLGetter, commons...)
+
+	if version != nil {
+		return &SentryProvider{
+			Version: version,
+			BaseUrl: baseURL,
+		}
+	} else {
```

Comment:
🚫 **[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`if` block ends with a `return` statement, so drop this `else` and outdent its block (golint)